src/main/resources/static/css/adminPage.css

```
@@ -0,0 +1,96 @@
+.page-container {
+    background-color: var(--lightest-blue);
+}
+
+/* Admin Pending sightings */
+
+.sightings-pending-container {
+    color: white;
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 500;
+    background: var(--darker-blue);
+    width: 100%;
+    padding: 1rem;
+}
+
+.sightings-pending-container h1{
+      text-align: center;
+      font-size: 2rem;
+      font-weight: 800;
+      padding: 1rem;
+}
+
+.sightings-pending-admin-form {
+    border: 5px solid var(--lightest-blue);
+    border-radius: 25px;
+    padding: 3rem 1rem;;
+    margin: 4rem 1rem 1rem;
+}
+
+.sighting {
+    display: inline-block;
+    padding-right: 4rem;
+}
+
+.sighting span{
+    padding: 2rem;
+    display: inline-block;
+}
+
+.button-submit-sights input[type="submit"]{
+    background-color: var(--medium-blue);
+    color: var(--darker-blue);
+    font-size: 2rem;
+    padding: 1rem 3rem;
+    border:none;
+    border-radius:10px;
+    position: relative;
+}
+
+.button-submit-sights input[type="submit"]:hover {
+  background-color: var(--darkest-blue);
+  color: white;
+}
+
+/* radio button to validate sight */
+
+.checkboxgroup input[type="radio"]{
+    height:20px;
+    width:20px;
+    margin-top: 30px;
+}
+
+.checkboxgroup {
+   display: inline-block;
+   text-align: center;
+   padding: 1rem;
+ }
+.checkboxgroup label {
+ display: block;
+}
+
+.flex-container-for-submit {
+    display: flex;
+    justify-content: right;
+}
+
+@media screen and (max-width: 768px) {
+    .checkboxgroup input[type="radio"]{ height:15px; width:15px; margin-top: 10px;}
+    .sightings-pending-container h1 {font-size: 1.5rem; font-weight:600;}
+    .sightings-pending-container {font-size:1rem;}
+    .sightings-pending-admin-form {padding:2rem;}
+    .sighting {padding-right: 0px;}
+    .button-submit-sights input[type="submit"] {font-size: 1rem; padding: 1rem;}
+    .flex-container-for-submit {justify-content: center;}
```

We probably want these `{..; ...; ...;}` in newlines, it will make reading diffs and handling merges a lot easier in future

```suggestion
    .checkboxgroup input[type="radio"] {
        height:15px; 
        width:15px; 
        margin-top: 10px;
    }
    .sightings-pending-container h1 {
        font-size: 1.5rem; 
        font-weight:600;
    }
    .sightings-pending-container {
        font-size:1rem;
    }
    .sightings-pending-admin-form {
        padding:2rem;
    }
    .sighting {
        padding-right: 0px;
    }
    .button-submit-sights input[type="submit"] {
        font-size: 1rem; padding: 1rem;
    }
    .flex-container-for-submit {
        justify-content: center;
    }
```

---

Thank you. I will know now for next time how to write clear CSS :)